src/stores/app-store.ts

```diff
@@ -1,11 +1,15 @@
 import { useState } from 'react';
 import constate from 'constate';
+import { initialFeatures } from '../data/api/datatyper/feature';
 
 export const [AppStoreProvider, useAppStore] = constate((initalValues: { fnr: string }) => {
     const [fnr, setFnr] = useState(initalValues.fnr);
+    const [features, setFeatures] = useState(initialFeatures);
```

Comment:
Vi gjør ingenting med `features` etter at den settes til default-verdien (`initialFeatures`). Vi må også faktisk kalle features-endepunktet og oppdatere `features`; dette er tilsvarende det som gjøres i `FeatureFetcher`-komponenten i **veilarbmaofs**.

Reply:
Er usikker på om jeg har gjort rett i de siste endringene mine for å fikse dette, kan du se over igjen nå?